xmodule/capa/safe_exec/tests/test_safe_exec.py

```diff
@@ -370,32 +370,63 @@ def remote_exec(data):
         assert isinstance(results['raised'], SafeExecException)
         assert 'whatever.py' in repr(results['raised'])
 
+    def test_default_normalizers(self):
```

Comment:
Let me know if you think this is too complicated, but ideally, if this is to test all the default normalizers, you could also loop through and test with all combos of n-1 default normalizers, and ensure that it fails for all those combos, proving out the necessity of each normalizer and that your test case is testing them all.

Reply:
In this case I really just wanted to test some of the more complicated ones and make sure I got the slashes right -- it doesn't need to be totally comprehensive.